flytekit/models/literals.py

```diff
@@ -698,11 +586,131 @@ def from_flyte_idl(cls, pb2_object):
         :param flyteidl.core.literals_pb2.LiteralMap pb2_object:
         :rtype: LiteralMap
         """
-        return cls({k: Literal.from_flyte_idl(v) for k, v in pb2_object.literals.items()})
+        return cls({k: Literal.from_flyte_idl(v) for k, v in _six.iteritems(pb2_object.literals)})
+
+
+class Scalar(_common.FlyteIdlEntity):
```

Comment:
why is this here?

Reply:
nm it just moved

Reply:
the new file will be identical to master.  not sure what you mean, how did these files ever get here?  Or why is scalar defined here?